### PR TITLE
Update app.py

### PR DESCRIPTION
**Pull request by alexdannyvaladares**
DummyTest

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def add():
     add_expense(description, amount)
     return redirect(url_for('index'))
 
+
```
Comment (ruialmeida51) [inline app.py:28]: Maybe we don't need this space.

File: app.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 """
 Created on Sat Apr  6 14:49:46 2024
 
-@author: Alex
+@author: Alex Valadares
```
Comment (ruialmeida51) [inline app.py:5]: The author is not the best.

**Review by ruialmeida51 — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (alexdannyvaladares) [reply app.py:5]: No

Comment (ruialmeida51) [reply app.py:5]: Understandable. Closing.

**Review by ruialmeida51 — APPROVED** [review]

**Pull request merged**